File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/SetAwesomeUrl.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class SetAwesomeUrl{

	@SerializedName("baseUrl")
	private String baseUrl;

	@SerializedName("elapsedMediaTimeSeconds")
	private int elapsedMediaTimeSeconds;

	public void setBaseUrl(String baseUrl){
		this.baseUrl = baseUrl;
	}

	public String getBaseUrl(){
		return baseUrl;
	}

	public void setElapsedMediaTimeSeconds(int elapsedMediaTimeSeconds){
		this.elapsedMediaTimeSeconds = elapsedMediaTimeSeconds;
	}

	public int getElapsedMediaTimeSeconds(){
		return elapsedMediaTimeSeconds;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/SetAwesomeUrl.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class SetAwesomeUrl {
    @SerializedName("baseUrl")
    var baseUrl: String? = null

    @SerializedName("elapsedMediaTimeSeconds")
    var elapsedMediaTimeSeconds = 0
    override fun toString(): String {
        return "SetAwesomeUrl{" +
                "baseUrl = '" + baseUrl + '\'' +
                ",elapsedMediaTimeSeconds = '" + elapsedMediaTimeSeconds + '\'' +
                "}"
    }
}